content/learn/quick-start/getting-started/setup.md

```diff
@@ -143,84 +143,118 @@ You might think to simply develop in release mode instead, but we recommend agai
 
 ### Enable Fast Compiles (Optional)
 
-Bevy can be built just fine using default configuration on stable Rust. However for maximally fast iterative compiles, we recommend the following configuration:
+Bevy can be built just fine using default configuration on stable Rust.
+Unfortunately, the compile times are rather long.
+This section explains how to speed up iterative compiles: the amount of time it takes to rebuild your project after changing a single file.
 
-* **Enable Bevy's Dynamic Linking Feature**: This is the most impactful compilation time decrease! If `bevy` is a dependency, you can compile the binary with the "dynamic_linking" feature flag (enables dynamic linking). **Important!** On Windows you _must_ also enable the [performance optimizations](#compile-with-performance-optimizations) or you will get a [`too many exported symbols`](https://github.com/bevyengine/bevy/issues/1110#issuecomment-1312926923) error.
+#### Dynamic Linking
+
+This is the most impactful compilation time decrease!
+If `bevy` is a dependency, you can compile the binary with the `dynamic_linking` feature flag.
+
+```sh
+cargo run --features bevy/dynamic_linking
+```
+
+If you don't want to add the `--features bevy/dynamic_linking` to each run, this flag can permanently be set via `Cargo.toml`:
+
+```toml
+[dependencies]
+bevy = { version = "0.13.0", features = ["dynamic_linking"] }
+```
+
+{% callout(type="note") %}
+Shipping your game with dynamic linking enabled is not recommended.
+Otherwise you will need to include `libbevy_dylib` alongside your game if you want it to run.
```

Comment:
```suggestion
Shipping your game with dynamic linking enabled is not recommended because it requires you to include `libbevy_dylib` alongside your game, it prevents certain optimizations, and can increase the size of your game.
```

I tested it; compiling with dynamic linking adds about 30MB to your game if you sum the size of the main executable and `libbevy_dylib`.